### ISSUE TITLE
Good work

### ISSUE THREAD

**Issue by Jh0nW1cK** [loomne/AutoJunk_cpp_csharp#7]
Good work friend, 
can you improve it so that it also adds garbage at the beginning and in the middle of the code?

**Comment by loomne** [comment]
Yes. it is planned for new autojunk called autojunk hero. It will be released after version 0.5.9.
